app/routes/routes.py

```diff
@@ -0,0 +1,266 @@
+from flask import redirect, render_template, request, session, url_for
+from flask_login import (UserMixin, current_user, login_required, login_user,
+                         logout_user)
+
+from app import app, cipher_suite, login_manager
+from app.services.image_upload import get_image
+from app.services.geolocalization_services.user_localization_and_distance import (country_name_to_code,
+                                                generate_random_ip, get_cities,
+                                                get_location, get_coordinates)
+from models import RobotProfile, User, UserRobot
+from services.database_operations.database_operations import (get_user_from_User_table,
+                                        if_user_is_admin,
+                                        insert_into_robots_db,
+                                        insert_user_and_user_profile,
+                                        select_all_from_database,
+                                        user_in_database,
+                                        update_user_location)
+from forms.forms import (AdminForm, LoginForm, RegistrationForm, UploadForm,
+                          UserCriteria, UserProfileForm)
+
+from app import bcrypt
+
+class User(UserMixin):
+    pass
+
+site_session = session
+
+@app.route("/")
+@app.route("/home")
+def home():
+    # print(get_location())
+    # print(country_name_to_code())
+    #print(get_cities())
+    # x = country_name_to_code("Poland")
+    # print(x)
+    # print(get_cities(x))

```

Comment:
Next time, before a code review, please make sure that the code is clean:
- it does not have redundant comments
- you've run black and isort
- you've removed unnecessary files from the repo
- etc..

It really makes a difference for a reviewer